benchmarks/src/test/java/org/apache/druid/benchmark/BenchmarkDataGeneratorTest.java

```diff
@@ -431,21 +432,11 @@ public void addRow(InputRow row)
     public void printStuff()
     {
       System.out.println();
-      for (String dim : dimensionMap.keySet()) {
-        System.out.println("DIMENSION " + dim + "\n============");
-        Map<Object, Integer> valueMap = dimensionMap.get(dim);
-
-        List<Comparable> valList = new ArrayList<>();
-        for (Object val : valueMap.keySet()) {
-          valList.add((Comparable) val);
-        }
+      for (Map<Object, Integer> valueMap : dimensionMap.values()) {
+        System.out.println("DIMENSION " + valueMap + "\n============");
```

Comment:
Now it prints `valueMap` twice. It should be `entrySet()` iteration (or better `dimensionMap.forEach()` call for readability).

Reply:
done